compiler/damlc/daml-stdlib-src/DA/Time.daml

```diff
@@ -53,11 +53,11 @@ hours : Int -> RelTime
 hours x = minutes (60 * x)
 
 -- | A number of minutes in relative time.
-minutes : Int -> RelTime
+minutes: Int -> RelTime
 minutes x = seconds (60 * x)
 
 -- | A number of seconds in relative time.
-seconds : Int -> RelTime
```

Comment:
Why are you moving further away from the style in the rest of the standard library? Please revert.

Reply:
My bad. :)